packages/wdio-sync/webdriverio-core.d.ts

```diff
@@ -813,6 +825,68 @@ declare namespace WebdriverIO {
         ): boolean;
     }
 
+    interface Network {
+        
+        /**
+         * Mock the response of a request. You can define a mock based on a matching
+         * glob and corresponding header and status code. Calling the mock method
+         * returns a stub object that you can use to modify the response of the
+         * web resource.
+         */
+        mock(
+            url: string,
+            filterOptions?: MockFilterOptions
+        ): Mock;
+
+        /**
+         * some description
+         */
+        throttle(): void;
+    }
+
+    interface Mock {
+        
+        /**
+         * Abort the request with an error code.
+         */
+        abort(
+            errorCode: ErrorCode
+        ): void;
+
+        /**
+         * Abort the request once with an error code.
+         */
+        abortOnce(
+            errorCode: ErrorCode
+        ): void;
+
+        /**
+         * Resets all information stored in the `mock.calls` array.
+         */
+        clear(): void;
+
+        /**
+         * Always respond with same overwrite.
+         */
+        respond(
+            overwrites: object | string,
```

Comment:
I guess we can change it to any? Or at least Record<string, unknown>, Array<Record<string, unknown>>, Function, string.

Reply:
actually this could be a function too, so all usages are as follows:

```js
mock.respond([{
    title: 'Injected Todo',
    order: null,
    completed: false,
    url: "http://todo-backend-express-knex.herokuapp.com/916"
}])
mock.respond('./node_modules/file.js')
mock.respond('https://test.com/some-image.png')
mock.respond((req) => {
    return req.body.map((item, i) => ({ ...item, title: i }))
})
```

So I will update it to `object | Function | string`